### ISSUE TITLE
Docked VS undocked

### ISSUE THREAD

**Issue by retiredit** [marticliment/ElevenClock#1301]
### Please confirm these before moving forward

- [X] I have searched for my feature proposal and not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template](https://github.com/marticliment/ElevenClock/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)

### Describe the new feature

I normally use my laptop in its dock with multiple monitors. When I travel it is stand alone with just the builtin screen. 
When I'm docked I have ElevenClock set so that I see the time and date on all monitors. When I'm in standalone mode I have to reset your app so that it appears on just 1 monitor. Can you have the app auto adjust for the mode it is being used in? Thanks

### Describe how this new feature could help users.

It really is just a convenience thing, a nice to see feature.

**Comment by marticliment** [comment]
> I have to reset your app

Can you please specify what you mean with "reset"?

**Comment by retiredit** [comment]
I have the display set to see the day of the week, it looks like this when I disconnect from the dock
[A screenshot of a clock  Description automatically ***@***.***


Sam
***@***.******@***.***>
“The main obstacle to progress is not ignorance but the illusion of knowledge.” Dr. Ron L. Graham
"We can evade reality, but we cannot evade the consequences of evading reality." Ayn Rand

“Waste no more time arguing what a good man should be. Be One.” – Marcus Aurelius

**Comment by marticliment** [comment]
Hello,

Attaching images through email does not post them on GitHub, therefore I can't see them.

Could you please attach them directly on the forum?

Thank you!

**Comment by retiredit** [comment]
I will but it will have to be later today as I’m no longer  near a computer

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by retiredit** [comment]
<img width="89" alt="image001" src="https://github.com/marticliment/ElevenClock/assets/75337661/5ff53927-80ba-4e0d-95db-6c4dbc4282f2">
<img width="329" alt="image002" src="https://github.com/marticliment/ElevenClock/assets/75337661/1bad55a7-5625-4799-b7fe-57bf3f97aa2f">

**Comment by retiredit** [comment]
Can you see these? If not I’ll have to do it later on. Computer

**Comment by marticliment** [comment]
Yes, I can see them now.

Can you please try enabling the following setting?
<img width="841" alt="image" src="https://github.com/marticliment/ElevenClock/assets/53119851/9ebc464c-1f5f-4aa7-a0e2-77e96032ae38">

**Comment by retiredit** [comment]
I just enabled that option. What does that have to do with showing the day of week?

**Comment by marticliment** [comment]
Oh, I see. Did read the message to fast.
Can you please send me your ElevenClock logs?

**Comment by retiredit** [comment]
[Sam Log File.txt](https://github.com/marticliment/ElevenClock/files/13781524/Sam.Log.File.txt)

**Comment by marticliment** [comment]
Hello,

It seems as if ElevenClock was not detecting that you have that option selected.

Can you please share me your log when your laptop is docked? It does then show as expected, right?

**Comment by retiredit** [comment]
Yes, it does as expected. I won’t be connected to my dock until late tomorrow

And thank you for your help

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by retiredit** [comment]
As discussed here is the log file when my computer is docked, I am seeing the day of the week perfectly
[Aruti log docked.txt](https://github.com/marticliment/ElevenClock/files/13793469/Aruti.log.docked.txt)

**Comment by marticliment** [comment]
Hello, is have been looking at your logs and perhaps there is a misconfiguration in ElevenClock Settings.

Can you please try resetting ELevenClock and reconfiguring it, and see if this issue perists?

**Comment by retiredit** [comment]
Are you saying delete it and then re-install it?

**Comment by marticliment** [comment]
No, just reset the settings.

**Comment by retiredit** [comment]
Before I tell you what happened let me tell you about my configuration. I have a laptop that is connected to 2 monitors. The laptop is kept closed so that I only use 2 monitors, the laptop screen not being one of them.

I just opened the laptop cover so that I was looking at 3 screens. At the instant I opened the laptop the day of the week disappeared from your software and then within seconds it reappeared on all screens.

Next, I disconnected the 2 external monitors and again the day of the week disappeared and then within seconds reappeared.

And finally, I reconnected the external monitors and again the day of the week disappeared but within seconds reappeared.

The end result is that reinitializing your software fixed the issue and worked perfectly in different scenarios.

Thank you very much for all of your help. Have a great New Year and be safe.



















when
